stylesheets/utility.css

```diff
@@ -193,4 +193,10 @@ h5{
     width: 100%;
 }
 
+.fix-width-50{
+    width: 50rem;
+}
 
+.fix-width-30{
+    width: 30rem;
+}
```

Comment:
You can remove `fix` `width-30` is self-explanatory.

Reply:
okay